lib/index.js

```diff
@@ -146,7 +146,15 @@ exports.init = function(options, callback) {
                 }
             }
             if (data[item]) {
-                sorted[item] = data[item];
+                if (options.onlyunknown) {
+                    if (data[item].licenses && data[item].licenses !== UNKNOWN) {
```

Comment:
Doesn't this line negate the `onlyunknown` since if it matches `UNKNOWN` then it get's skipped?